RPIPs/RPIP-43.md

```diff
@@ -126,9 +126,8 @@ into shares is defined in [RPIP-46](RPIP-46.md).
 - Newly deposited capital that is awaiting deposit to the beacon chain SHALL be
   excluded from capital distribution, but rather be subject to separate functions
   for `prestake` and `stake` transactions.
-- The megapool MAY be used as the node operator's authorized fee recipient for
-  execution layer rewards if they are not opted into the smoothing pool. These
-  funds shall be subject to rewards distribution.
+- The megapool or the node distributor SHALL be allowed to be used as the node operator's authorized fee recipient for
```

Comment:
Must it be one of these two in this case? Or are other addresses also valid? If so, should be reworded.